lib/rules/color-hex-case/__tests__/index.js

```diff
@@ -164,3 +164,323 @@ testRule(
 		],
 	}),
 );
+
+testRule({
+	ruleName,
+	config: ['lower'],
+	syntax: 'css-in-js',
+	fix: true,
+	accept: [
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	stroke: "url(#gradiantA)",
+});
+`,
+			description: 'href with location',
+		},
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	color: "pink",
```

Comment:
```suggestion
	color: "PINK",
```

Let's use uppercase to make sure it's ignored.